www/class/centreonContactgroup.class.php

```diff
@@ -222,6 +222,27 @@ public function syncWithLdap()
             if (false != $connectionResult) {
                 $res = $this->db->query("SELECT cg_id, cg_name, cg_ldap_dn FROM contactgroup
                     WHERE cg_type = 'ldap' AND ar_id = " . $ldaprow['ar_id']);
+
+
+                /**
+                 * insert groups from ldap into centreon
+                 */
+                $registeredGroupsFromDB = $res->fetchAll();
+                $registeredGroups = [];
+                foreach ($registeredGroupsFromDB as $registeredGroupFromDB){
+                    $registeredGroups[] = $registeredGroupFromDB['cg_name'];
+                }
+                $ldapGroups = $ldapConn->listOfGroups();
+                $toInsertGroups = array_diff($ldapGroups, $registeredGroups);
+
+                foreach ($toInsertGroups as $toInsertGroup){
+                    $this->insertLdapGroup('['.$ldaprow['ar_id'].']'.$toInsertGroup);
+                }
+
+                $res = $this->db->query("SELECT cg_id, cg_name, cg_ldap_dn FROM contactgroup
+                    WHERE cg_type = 'ldap' AND ar_id = " . $ldaprow['ar_id']);
+
+
                 while ($row = $res->fetchRow()) {
```

Comment:
please use fetch() instead of fetchRow() occurrences, when working on a php7 project // 2.9.
fetchRow() is surcharged with fetch() in `centreon/class/centreonPdoStatement`

Reply:
thanks for the reminder, pushing that fix in a moment :)